tienvx/fos-oauth-server-bundle-orm/1.0/config/packages/fos_oauth_server_orm.yaml

```diff
@@ -0,0 +1,9 @@
+fos_oauth_server:
+    db_driver: orm       # Drivers available: orm, mongodb, propel or custom
+    client_class:        FOS\OAuthServerBundle\Entity\Client
+    access_token_class:  FOS\OAuthServerBundle\Entity\AccessToken
+    refresh_token_class: FOS\OAuthServerBundle\Entity\RefreshToken
+    auth_code_class:     FOS\OAuthServerBundle\Entity\AuthCode
+framework:
+    templating:
+        engines: ['twig']
```

Comment:
You cannot modify config from other bundles. 

Add instructions about this in a post-install.txt